tic-tac-toe/src/scripts/calculate-next-move.js

```diff
@@ -0,0 +1,91 @@
+function checkTurn(arr, own, opossit) {
+    const ownRate = [
+        checkDiagonal(arr, own, opossit, false, 10),
+        checkDiagonal(arr, own, opossit, true, 10),
+        checkLine(arr, own, opossit, 10, true),
+        checkLine(arr, own, opossit, 10, false)
+    ];
+    const opossitRate = [
+        checkDiagonal(arr, opossit, own, false, 1),
+        checkDiagonal(arr, opossit, own, true, 1),
+        checkLine(arr, opossit, own, 1, true),
+        checkLine(arr, opossit, own, 1, false)
+    ]
+    const nextTurn = [...ownRate, ...opossitRate].sort((a, b) => {
+        return b.rate - a.rate;
+    })[0].positions[0];
+    return nextTurn;
+}
+
+function checkDiagonal(arr, own, opossit, isLineGrow = false, rateMult = 1, ) {
+  const squareSize = 3;
+  let rate = 0;
+  let recomPosition = [];
+  for (let i = 0; i < squareSize; i++) {
+    const position = getDiagonalPosition(i, squareSize, isLineGrow);
+    const value = arr[position];
+    if (value === own) {
+      rate = rate > 0 ? 100 * 10 : 10;
```

Comment:
You should use variable instead of 10, 100

Reply:
I fixed this.
```suggestion
    if (value === firstPlayerValue) {
      rating = rating > 0 ? twoItemsCombinationCost * oneItemsCombinationCost : oneItemsCombinationCost;
    }
```